src/handlers/dns/DnsStreamHandler.h

```diff
@@ -112,6 +114,7 @@ class DnsMetricsBucket final : public visor::AbstractMetricsBucket
             , NOERROR("dns", {"wire_packets", "noerror"}, "Total DNS wire packets flagged as reply with return code NOERROR (ingress and egress)")
             , NODATA("dns", {"wire_packets", "nodata"}, "Total DNS wire packets flagged as reply with return code NOERROR and no answer section data (ingress and egress)")
             , filtered("dns", {"wire_packets", "filtered"}, "Total DNS wire packets seen that did not match the configured filter(s) (if any)")
+            , queryECS("dns", {"wire_packets", "query_ecs"}, "Total queries that have EDNS Client Subnet (ECS) field set")
```

Comment:
@leoparente Is the the "Total queries that..." or the "Total DNS wire packets that..."? Will it count both DNS queries and responses packets with ECS set?

Reply:
only queries @rboucher-ns1